### PR TITLE
Feature article for IPv6

### PR DESCRIPTION
**Pull request by alpeb**
~This also enables emojis for the entire site.~
Short and sweet.

### CODE REVIEW

File: linkerd.io/content/2.16/features/ipv6.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| Provider | Dual-Stack Supported | IPv6-only Supported |
+|----------|----------------------|---------------------|
+|    AKS   |  :white_check_mark:  |        :x:          |
+|    EKS   |         :x:          | :white_check_mark:  |
+|    GKE   |  :white_check_mark:  |        :x:          |
+|    Kind  |  :white_check_mark:  | :white_check_mark:  |
+|    k3d   |         :x:          |        :x:          |
+{{< /table >}} 
+
+
```
Comment (wmorgan) [inline linkerd.io/content/2.16/features/ipv6.md:41]: Thank you for writing these, but I suggest removing both these sections. We don't really need to advocate for IPv6 as a technology, and I don't want to maintain a compatibility table for cloud provider support.

File: linkerd.io/content/2.16/features/ipv6.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Linkerd IPv6 Configuration
+
+IPv6 support in Linkerd is disabled by default. To enable, just set
+`proxy.disableIPv6=false`. This setting is to be applied to the control plane
+and the linkerd-cni plugin, if used.
+
+This mode doesn't make any difference in how Linkerd works. Bear in mind however
+that when faced with a dual-stack service, the proxy will only forward traffic
+to the endpoints in the IPv6 EndpointSlice.
```
Comment (wmorgan) [inline linkerd.io/content/2.16/features/ipv6.md:50]: Enabling IPv6 support does not generally change how Linkerd operates, except in one way: when enabled on a dual-stack cluster, Linkerd will only use the IPv6 endpoints of destinations and will not use the IPv4 endpoints.

File: linkerd.io/hugo.toml [hunk 3]
```diff
@@ -6,6 +6,7 @@ languageCode = "en-US"
 sectionPagesMenu = "main"
 theme = "buoyant"
 title = "Linkerd"
+enableEmoji = true
```
Comment (wmorgan) [inline linkerd.io/hugo.toml:9]: Can be removed if you remove the sections above

File: linkerd.io/content/2.16/features/ipv6.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+|    Kind  |  :white_check_mark:  | :white_check_mark:  |
+|    k3d   |         :x:          |        :x:          |
+{{< /table >}} 
+
+
+## Linkerd IPv6 Configuration
+
+IPv6 support in Linkerd is disabled by default. To enable, just set
+`proxy.disableIPv6=false`. This setting is to be applied to the control plane
+and the linkerd-cni plugin, if used.
```
Comment (kflynn) [inline linkerd.io/content/2.16/features/ipv6.md:46]:
```suggestion
`proxy.disableIPv6=false` when installing the control plane and, if you use it, the linkerd-cni plugin.
```

??

**Review by adleong — APPROVED** [review]

**Pull request merged**